sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuParquetScan.scala

```diff
@@ -463,13 +463,17 @@ private case class GpuParquetFileFilterHandler(@transient sqlConf: SQLConf) exte
   private val pushDownDate = sqlConf.parquetFilterPushDownDate
   private val pushDownTimestamp = sqlConf.parquetFilterPushDownTimestamp
   private val pushDownDecimal = sqlConf.parquetFilterPushDownDecimal
-  private val pushDownStringStartWith = sqlConf.parquetFilterPushDownStringStartWith
+  // From Spark 340, more string predicates are supported as push-down filters, so this
+  // flag is renamed to 'xxxxStringPredicate' and specified by another config. Then shims
+  // are required.
```

Comment:
Not sure this comment adds much.  IMO it's obvious there are shims here because we're calling something with `Shims` in the name.

Reply:
Updated.